### PR TITLE
CI: Test all supported Python versions

### PR DESCRIPTION
**Pull request by incognitojam**
Expanding the supported Python versions is required for https://github.com/commaai/body-jim/issues/9

### CODE REVIEW

**Comment by incognitojam** [comment]
Just a weird issue with Cython 3.12... Seems to be PyAV https://github.com/PyAV-Org/PyAV/issues/1140#issuecomment-2136082607. Will try and reproduce and fix locally

**Comment by incognitojam** [comment]
@fredyshox is it ok to bump PyAV?

File: pyproject.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ classifiers = [
 dependencies = [
   "aiortc>=1.6.0",
   "aiohttp>=3.7.0",
-  "av>=9.0.0,<11.0.0",
+  "av>=11.0.0,<13.0.0",
```
Comment (adeebshihadeh) [inline pyproject.toml:21]: @fredyshox any reason for the less than?

Comment (fredyshox) [reply pyproject.toml:21]: not sure, its probably gonna be ok

**Pull request merged**